modules/menu/css/stories.tsx

```diff
@@ -1,22 +1,71 @@
-import React from 'react';
+import React, {Component} from 'react';
 import {storiesOf} from '@storybook/react';
 import withReadme from 'storybook-readme/with-readme';
 import README from './README.md';
-import {css} from 'emotion';
 import './index.scss';
+// @ts-ignore
+import initializeIcons from '../../icon/css/lib/canvas-kit-css-icon';
 
-const containerStyle = css({
-  paddingTop: '12px',
-});
+class WithIconsDemo extends Component {
+  componentDidMount() {
+    initializeIcons();
+  }
+
+  render() {
+    return (
+      <div className="wdc-type">
+        <div className="wdc-menu" style={{width: '250px'}}>
+          <ul>
+            <li className="wdc-menu-item-focused">
+              <i
+                className="wdc-icon wdc-menu-item-icon"
```

Comment:
Following up on our slack convo earlier: technically, I could remove the `wdc-menu-item-icon` class and just target `wdc-icon`. Or do you want me to leave it there in case we can't safely assume the icon will have a `wdc-icon`class?

Reply:
Sounds good, I've updated the README accordingly.